shared/src/main/scala/amf/core/client/scala/config/RenderOptions.scala

```diff
@@ -14,7 +14,7 @@ case class RenderOptions(
     filterFields: Field => Boolean = (_: Field) => false,
     amfJsonLdSerialization: Boolean = true,
     useJsonLdEmitter: Boolean = false,
-    flattenedJsonLd: Boolean = false,
+    private[amf] val flattenedJsonLd: Boolean = false,
```

Comment:
Are we always calling the flattened jsonld emitter directly?

Reply:
If that is so, we can delete this render option directly. If not shouldn't it be `true` by default?

Reply:
I left it there to use it from the testing module. Yup, should be true by default.

Reply:
Fixed